.github/workflows/suite-native_production_ci.yml

```diff
@@ -1,23 +1,44 @@
-#name: "[Build] release"
-#
-#on: [pull_request]
-#jobs:
-#  build:
-#    name: Install and build
-#    runs-on: ubuntu-latest
-#    steps:
-#      - uses: actions/checkout@v3
-#      - uses: actions/setup-node@v3
-#        with:
-#          node-version-file: ".nvmrc"
-#          cache: yarn
-#      - name: Setup EAS
-#        uses: expo/expo-github-action@v8
-#        with:
-#          eas-version: latest
-#          token: ${{ secrets.EXPO_TOKEN }}
-#      - name: Install libs
-#        run: yarn install
-#      - name: Build on EAS
-#        run: eas build --platform all --profile production --auto-submit --non-interactive
-#        working-directory: suite-native/app
+name: "[Release] suite-native production"
+
+on:
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    name: Install and build
```

Comment:
This should also be creating environment release for release approvement to work
```yaml
  build:
    name: Install and build
    environment: production
```
cc @matejkriz

Reply:
cc99ee1f2041b704239b27d05f9fbb69f77eee57